frontend/src/components/form/InputNumber.vue

```diff
@@ -0,0 +1,62 @@
+<script setup lang="ts">
+import { toRef, ref } from 'vue';
+import { useField, ErrorMessage } from 'vee-validate';
+
+import { InputNumber } from '@/lib/primevue';
+
+import type { Ref } from 'vue';
+
+// Props
+type Props = {
+  helpText?: string;
+  label?: string;
+  name: string;
+  placeholder?: string;
+  disabled?: boolean;
+  bold?: boolean;
+};
+
+const props = withDefaults(defineProps<Props>(), {
+  helpText: '',
+  type: 'text',
+  label: '',
+  placeholder: '',
+  disabled: false,
+  bold: true
+});
+
+// State
+const { errorMessage, value } = useField<number>(toRef(props, 'name'));
+const fieldActive: Ref<boolean> = ref(false);
+</script>
+
+<template>
+  <div class="field col">
+    <label
+      :class="{ 'font-bold': bold }"
+      :for="name"
+    >
+      {{ label }}
+    </label>
+    <InputNumber
+      v-model.trim="value"
+      :aria-describedby="`${name}-help`"
+      :name="name"
+      :placeholder="placeholder"
+      class="w-full"
+      :class="{ 'p-invalid': errorMessage }"
+      :disabled="disabled"
+      @focus="fieldActive = true"
+      @blur="fieldActive = false"
+    />
+    <small
+      v-if="fieldActive"
+      :id="`${name}-help`"
+    >
+      {{ helpText }}
+    </small>
+    <div>
+      <ErrorMessage :name="name" />
+    </div>
```

Comment:
Is the new div wrapper addressing some kind of strange spacing/layout issue? Usually we try to avoid nesting more DOM elements whenever possible.